script/Arcueid/Day 8/08_00_ARC08_6.txt

```diff
@@ -396,8 +396,11 @@ We're going to be looking for vampires just like yesterday, right?"
 [sha:0b11178794ccc7baa73df28dc8847a39891c2b50]{
 -- Page 317, Offset 20460.
 -- 夜の街を散策する。
-// Please stop using 夜の街 it is so annoying to TL
-We stroll through the nightlife district.
+// Chatting with Petrikow about 夜の街 before, he basically said that
+// 1. this isn't nightlife district, that's a mis-TL on my part.
+// 2. we needn't be quite so rigid with it.
+// alt suggestion: We stroll through the city's evening glow.
```

Comment:
City's evening glow makes sense to me if there's neon or something in the CG here (as in, the city _is_ glowing), otherwise I'd go for `twilit city` or something